package-lock.json

```diff
@@ -5496,6 +5505,7 @@
       "resolved": "https://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
       "integrity": "sha512-e8tQYnZodmebYDWGH7KMRvtzKXaJHx3BbilrgZCfvyLUYdKpK1t5PSPmpkny/SgiTSCnjfLW7v5rlONXVFkQEA==",
       "dev": true,
+      "optional": true,
```

Comment:
Can you remove all these little changes to `package-lock.json` and just keep the above change where the new package is added?

Reply:
Yep, sure thing!

`package-lock.json` may be worth a mention in the contributor guide 😅